cli/azd/cmd/env.go

```diff
@@ -76,6 +76,12 @@ func envActions(root *actions.ActionDescriptor) *actions.ActionDescriptor {
 		DefaultFormat:  output.EnvVarsFormat,
 	})
 
+	group.Add("rm", &actions.ActionDescriptorOptions{
```

Comment:
Is there a precedence in azd to shorten commands names linux style like this?   I think we should discuss if we want "rm" or "remove"